autotest/utils/pipeline_chat.py

```diff
@@ -57,18 +60,22 @@ def run_pipeline_chat_test(config,
     gen_config = GenerationConfig(top_k=1)
 
     config_log = os.path.join(
-        log_path,
-        'pipeline_config_' + type + model_case.split('/')[1] + '.log')
+        log_path, '_'.join([
+            'pipeline', 'config', type, worker_id,
+            model_case.split('/')[1] + '.log'
+        ]))
     file = open(config_log, 'w')
-    file.writelines(' '.join([
-        'reproduce config info:', hf_path,
-        str(backend_config),
-        str(gen_config)
+    file.writelines('\n'.join([
+        'reproduce config info:', 'engine_config = ' + str(backend_config),
+        'gen_config = ' + str(gen_config),
+        'pipe = pipeline("' + hf_path + '",  backend_config=engine_config)',
+        'res = pipe("Hi, pls introduce shanghai", gen_config=gen_config)'
     ]))
-    print(' '.join([
-        'reproduce config info:', hf_path,
-        str(backend_config),
-        str(gen_config)
+    print('\n'.join([
```

Comment:
why need to print it? if for logging, could we create a variable to this long text and use it for file logging and print

Reply:
> why need to print it? if for logging, could we create a variable to this long text and use it both for file logging and printing

help me to reproduce it  if the testcase fails. I can try to use a logger to log it.

Reply:
fixed!